src/backend/aspen/workflows/pangolin/save.py

```diff
@@ -33,7 +39,7 @@ def cli(pangolin_fh: io.TextIOBase, pangolin_last_updated: datetime):
         taxon_to_pango_info: Mapping[int, Mapping[str, Union[str, float]]] = {
             int(row["taxon"]): {
                 "lineage": row["lineage"],
-                "probability": get_probability(float(row["conflict"])),
+                "probability": get_probability(float(row["ambiguity_score"])),
```

Comment:
will this always be set?

Reply:
looks like in the docs it also might be blank https://cov-lineages.org/pangolin_docs/output.html

Reply:
in which case we might want to just add 
```
"probability": get_probability(float(row["ambiguity_score"])) if row["conflict"] != "" else 1,
```
so it doesn't error on the float cast